jwt_verify_lib/status.h

```diff
@@ -155,6 +155,14 @@ enum class Status {
   JwksX509ParseError,
   // X509 get pubkey fails
   JwksX509GetPubkeyError,
+
+  // Key type is not supported.
+  PKCSNotImplementedKty,
```

Comment:
should we prefix it with Pkcs8?  to be consistent.

Reply:
done